libraries/botbuilder-dialogs-adaptive/src/actions/case.ts

```diff
@@ -15,7 +15,7 @@ import { ActionScope } from './actionScope';
  */
 export class Case extends ActionScope {
     /**
-     * Initializes a new instance of the `Case` class.
+     * Initializes a new instance of the [Case](xref:botbuilder-dialogs-adaptive.Case) class.
      * @param value Optional. Case's string value.
      * @param actions Optional. Numerable list of dialog actions.
```

Comment:
Should we use the Dialog class reference here?, more lines with this point on this file.